src/ReduxStore/FeedDetails/feedSagas.js

```diff
@@ -1,30 +1,55 @@
-import { takeLatest, put, call } from "redux-saga/effects";
-import { INITIALDATA_LOADED, articleDataLoaded } from "./feedActions";
-import axios from "axios";
+import { fetchInitCommentData } from './fetchInitCommentData'
+import { takeLatest, put, call } from 'redux-saga/effects';
+import {
+  INITIALDATA_LOADED,
+  articleDataLoaded,
+  tagsDataLoaded
+} from "./feedActions";
+
 
 export const feedSaga = function*() {
   yield takeLatest(INITIALDATA_LOADED, loadInitialData);
 };
 
 const loadInitialData = function*() {
-  const initData = yield call(fetchData);
-  yield put(articleDataLoaded(initData.data["articles"]));
+  const initArticData = yield call(fetchInitArticleData);
+  console.log(initArticData);
+
+  yield put(articleDataLoaded(initArticData["articles"]));
+
+  const initTagData = yield call(fetchInitTagesData);
+  yield put(tagsDataLoaded(initTagData["tags"]));
+
+  const initCommentData = yield call(fetchInitCommentData);
+  console.log(initCommentData);
```

Comment:
`console.log` should be removed before PR.

Reply:
fixed